packages/app-utils/src/components/AppBootstrap.test.tsx

```diff
@@ -173,8 +173,8 @@ it('should log in automatically when the anonymous handler is supported', async
   });
 
   expect(screen.queryByTestId('auth-base-loading')).toBeNull();
-  expect(screen.queryByTestId('connection-bootstrap-loading')).not.toBeNull();
-  expect(screen.queryByText(mockChildText)).toBeNull();
+  expect(screen.queryByTestId('connection-bootstrap-loading')).toBeNull();
+  expect(screen.queryByText(mockChildText)).not.toBeNull();
```

Comment:
The test shouldn't need any changes. The test was failing because of a change in behavior. With the other comments about loading being shown when connecting, the test passes as it was originally written